config/webpack.dev.config.js

```diff
@@ -13,31 +15,35 @@ module.exports = Merge.smart(commonConfig, {
     path.resolve(__dirname, '../src/index.jsx'),
   ],
   module: {
+    // Specify file-by-file rules to Webpack. Some file-types need a particular kind of loader.
     rules: [
+      // The babel-loader transforms newer ES2015+ syntax to older ES5 for older browsers.
+      // Babel is configured with the .babelrc file at the root of the project.
       {
         test: /\.(js|jsx)$/,
         include: [
           path.resolve(__dirname, '../src'),
         ],
         loader: 'babel-loader',
         options: {
-          cacheDirectory: true,
+          cacheDirectory: true, // caches transforms to the filesystem for faster re-builds
```

Comment:
Can you provide more clarity here?

Reply:
Sure, I tried to add more detail to this comment (pulling from [babel-loader's own description of the option]